code/game/objects/random/biogenerator_randoms.dm

```diff
@@ -0,0 +1,82 @@
+// This list of randoms is intended for use in the biogenerator.
+// These ones are sanitized of anything funky compared to other random item lists
+
+/obj/random/biogenerator/backpack
+	name = "random backpack"
+	icon = 'icons/obj/storage/backpack.dmi'
+	icon_state = "backpack"
+	spawnlist = list(
+		/obj/item/storage/backpack,
+		/obj/item/storage/backpack/medic,
+		/obj/item/storage/backpack/security,
+		/obj/item/storage/backpack/industrial,
+		/obj/item/storage/backpack/hydroponics,
+		/obj/item/storage/backpack/pharmacy,
+		/obj/item/storage/backpack/satchel,
+		/obj/item/storage/backpack/toxins,
+		/obj/item/storage/backpack/satchel/leather,
+		/obj/item/storage/backpack/satchel/eng,
+		/obj/item/storage/backpack/satchel/med,
+		/obj/item/storage/backpack/satchel/pharm,
+		/obj/item/storage/backpack/satchel/sec,
+		/obj/item/storage/backpack/satchel/hyd,
+		/obj/item/storage/backpack/satchel/tox,
+		/obj/item/storage/backpack/duffel,
+		/obj/item/storage/backpack/duffel/hyd,
+		/obj/item/storage/backpack/duffel/med,
+		/obj/item/storage/backpack/duffel/eng,
+		/obj/item/storage/backpack/duffel/sec,
+		/obj/item/storage/backpack/duffel/pharm,
+		/obj/item/storage/backpack/duffel/tox,
+		/obj/item/storage/backpack/messenger,
+		/obj/item/storage/backpack/messenger/pharm,
+		/obj/item/storage/backpack/messenger/med,
+		/obj/item/storage/backpack/messenger/engi,
+		/obj/item/storage/backpack/messenger/hyd,
+		/obj/item/storage/backpack/messenger/sec,
+		/obj/item/storage/backpack/messenger/tox
+	)
+/obj/random/biogenerator/gloves
+	name = "random gloves"
+	icon = 'icons/obj/clothing/gloves.dmi'
+	icon_state = "yellow"
+	spawnlist = list(
+		/obj/item/clothing/gloves/black,
+		/obj/item/clothing/gloves/black/tajara,
+		/obj/item/clothing/gloves/black/unathi,
+		/obj/item/clothing/gloves/black_leather,
+		/obj/item/clothing/gloves/black_leather/tajara,
+		/obj/item/clothing/gloves/black_leather/unathi,
```

Comment:
Same comment as for the random backpacks.
I think this list is way too big and random to be useful.